src/status_im/ui/components/toolbar/view.cljs

```diff
@@ -79,12 +79,16 @@
 (defn content-title
   ([title] (content-title nil title))
   ([title-style title]
-   (content-title title-style title nil nil))
-  ([title-style title subtitle-style subtitle]
+   (content-title title-style title nil nil nil))
+  ([title-style title subtitle-style subtitle additional-text-props]
    [react/view {:style styles/toolbar-title-container}
-    [react/text {:style (merge styles/toolbar-title-text title-style)
-                 :font  :toolbar-title}
-     title]
+    (if platform/ios?
+      [react/text (merge {:style (merge styles/toolbar-title-text title-style)
+                          :font  :toolbar-title :adjustsFontSizeToFit true :numberOfLines 1}
+                         additional-text-props) title]
+      [react/text (merge {:style (merge styles/toolbar-title-text title-style)
```

Comment:
```
[react/text (merge {:style (merge styles/toolbar-title-text title-style)
                                :font  :toolbar-title :numberOfLines 1}
                                (if platform/ios? {:adjustsFontSizeToFit true} {:ellipsizeMode :tail}
                                 additional-text-props) title]
```

Reply:
yes, more compact version. I'll update it shortly.